webapp/commonComponents/form/checkbox.js

```diff
@@ -16,7 +16,7 @@ class Checkbox extends React.Component {
 
     return (
       <div style={{ justifySelf: 'start' }}>
-        <ValidationTooltip validation={validation}>
+        <ValidationTooltip validation={validation} showKeys={false}>
```

Comment:
checkbox is generic. should showKeys be passed from the caller?

Reply:
In fact showKeys is not needed in this case, as there's no 'field' attribute in the validation object, I removed showKeys in <validationTooltip />